src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1044,4 +1043,40 @@ This version used in: {2}</value>
   <data name="Rename_type_to_0" xml:space="preserve">
     <value>Rename type to {0}</value>
   </data>
+  <data name="All_lowercase" xml:space="preserve">
+    <value>All lowercase</value>
+  </data>
+  <data name="All_uppercase" xml:space="preserve">
+    <value>All uppercase</value>
+  </data>
+  <data name="Begins_with_0" xml:space="preserve">
+    <value>Begins with '{0}'</value>
+  </data>
+  <data name="Begins_with_0_camel_case" xml:space="preserve">
+    <value>Begins with '{0}'; camel case</value>
+  </data>
+  <data name="Camel_Case" xml:space="preserve">
+    <value>Camel Case</value>
+  </data>
+  <data name="Ends_with_0" xml:space="preserve">
+    <value>Ends with '{0}'</value>
+  </data>
+  <data name="First_word_capitalized" xml:space="preserve">
+    <value>First word capitalized</value>
+  </data>
+  <data name="Pascal_Case" xml:space="preserve">
+    <value>Pascal Case</value>
+  </data>
+  <data name="Private_or_internal_field" xml:space="preserve">
+    <value>Private or internal field</value>
+  </data>
+  <data name="Private_or_internal_static_field" xml:space="preserve">
+    <value>Private or internal static field</value>
+  </data>
+  <data name="Public_or_protected_field" xml:space="preserve">
+    <value>Public or protected field</value>
```

Comment:
what about someone who wants to do something for public, protected and internal fields?